closed/NVIDIA/code/stable-diffusion-xl/tensorrt/download_model.sh

```diff
@@ -22,51 +22,61 @@ then
 else
     echo "WARNING: Please enter the MLPerf container (make prebuild) before downloading SDXL model."
     echo "WARNING: SDXL model is NOT downloaded! Exiting..."
-    exit 1
+    #exit 1
 fi
 
-MODEL_DIR=/work/build/models
-DATA_DIR=/work/build/data
+MODEL_DIR=${MLPERF_SCRATCH_PATH}/models
+DATA_DIR=${MLPERF_SCRATCH_PATH}/data
 
 # Download the fp16 raw weights of MLCommon hosted HF checkpoints
-download_file models SDXL/official_pytorch/fp16 \
-    https://cloud.mlcommons.org/index.php/s/LCdW5RM6wgGWbxC/download \
-    stable_diffusion_fp16.zip
+# Check whether the folder already exists, if yes then do not download
+if [ ! -d "${MODEL_DIR}/SDXL/official_pytorch/fp16" ]; then
+    echo "Directory not found, proceeding with download and verification..."
```

Comment:
In CM workflow this condition should never happen right? (As we create softlink)